eppo_core/src/ufc/models.rs

```diff
@@ -3,23 +3,33 @@ use std::collections::HashMap;
 use derive_more::From;
 use serde::{Deserialize, Serialize};
 
-use crate::{client::AssignmentValue, rules::Rule};
+use super::AssignmentValue;
 
-/// Universal Flag Configuration.
-#[derive(Debug, Serialize, Deserialize)]
+#[allow(missing_docs)]
+pub type Timestamp = chrono::DateTime<chrono::Utc>;
+
+/// Universal Flag Configuration. This the response format from the UFC endpoint.
+#[derive(Debug, Serialize, Deserialize, Clone)]
 #[serde(rename_all = "camelCase")]
 pub struct UniversalFlagConfig {
-    // Value is wrapped in `TryParse` so that if we fail to parse one flag (e.g., new server
-    // format), we can still serve other flags.
+    /// Flags configuration.
+    ///
+    /// Value is wrapped in `TryParse` so that if we fail to parse one flag (e.g., new server
```

Comment:
I really like this. Ideally, all our SDKs were this resilient. If only we had a way to easily get improvements like this to all the other SDKs as we think of them